vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -60,9 +60,9 @@ IMAGE_LOCATION: "{{ lookup('env', 'IMAGE_LOCATION') | default('https://artifacto
 IMAGE_URL: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}"
 IMAGE_CHECKSUM: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}.md5sum"
 
-IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2', true)}}"
+IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CENTOS_8.2_NODE_IMAGE_K8S_v1.18.8.qcow2', true)}}"
 RAW_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64-raw.img', true)}}"
```

Comment:
Should this also be changed to the similar name as the default value of IMAGE_NAME_CENTOS?